src/features/steps/SIMCard9151.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { Back } from '../../common/Back';
+import Link from '../../common/Link';
+import Main from '../../common/Main';
+import { Next } from '../../common/Next';
+
+export default () => (
+    <Main>
+        <Main.Content heading="Activate iBasis SIM card">
+            <p>There are two pre-activated SIM cards in the kit:</p>
+            <br />
+            <ol className="tw-list-outside tw-list-decimal tw-pl-4">
+                <li>
+                    <Link
+                        label="SIM card from onomondo"
+                        href="https://onomondo.com/go/nordic-dev-kit/#form"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 10MB free data for 30 days.
+                    <br />
+                    If you register the SIM card you will get additional 40MB
+                    and extend the trial period for 60 days and access the
+                    real-time network insight tool.
+                </li>
+                <br />
+                <li>
+                    <Link
+                        label="SIM card from Wireless Logic"
+                        href="https://www.wirelesslogic.com/simclaim/nsctrial/"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 5MB free data for 30 days.
+                    <br />
+                    If you register the SIM card you will get additional 45MB
+                    and extend the trial period for 1 year.
+                </li>
+            </ol>
+            <br />
+            <p>
+                Select the SIM card which is supported in your area (check the
+                links), plug in and it will work out of the box.
```

Comment:
```suggestion
                Select the SIM card that is supported in your area (check the
                links), plug it in, and it will work out of the box.
```